src/index.html

```diff
@@ -17,11 +21,360 @@
+        <img
+          class="card__icon"
+          src="images/imac.jpeg"
+          alt="card--icon"
+        />
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__rate-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-money">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__icon"
+          src="images/imac.jpeg"
+          alt="card--icon"
+        />
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__rate-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-money">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__icon"
+          src="images/imac.jpeg"
+          alt="card--icon"
+        />
+        <p class="card__info">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__rate-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-money">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```